 private void handleException(java.lang.Throwable exception) { handleUnexpectedException(exception); } 